testcases/manifests/error_multiple_resources_all_invalid.yaml

```diff
@@ -0,0 +1,52 @@
+# {
+#   "metadata": {},
+#   "status": "Failure",
+#   "message": " \"\" is invalid: spec.contAIN3rz: Invalid value: value provided for unknown field",
+#   "reason": "Invalid",
+#   "details": {
+#     "causes": [
+#       {
+#         "reason": "FieldValueInvalid",
+#         "message": "Invalid value: value provided for unknown field",
+#         "field": "spec.contAIN3rz"
+#       }
+#     ]
+#   },
+#   "code": 422
+# }
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-pod
+spec:
+  contAIN3rz:
+    - name: nginx
+      image: nginx
+      ports:
+        - protocol: 22
+          containerPort: 88
+---
+# {
```

Comment:
does validate_test.go not have to be modified to support multiple documents/properly pick up this comment?

Reply:
Done